index.css

```diff
@@ -56,6 +56,13 @@ img {
 @media (max-width: 900px) {
   .middle-container {
     flex-direction: column;
+    height: 40%;

```

Comment:
note to self:
Before merge, fix height property value. Not sure this is really doing anything meaningful.

Reply:
Changes now complete.